assets/scripts/dialogs/Analytics/SegmentAnalytics.js

```diff
@@ -1,81 +1,81 @@
-import React, { useState } from 'react'
+import React from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import SegmentForPalette from '../../segments/SegmentForPalette'
 import { FormatNumber } from '../../util/formatting'
-import Icon from '../../ui/Icon'
+import { FormattedMessage } from 'react-intl'
+import { getLocaleSegmentName } from '../../segments/view'
 
-const generateClass = (value) => {
-  if (value > 12000) {
-    return 'positive'
-  }
-
-  if (value > 5000) {
-    return 'warning'
-  }
+const BAR_HEIGHT = '70px'
+const BAR_MODIFIER = 0.85
+const BAR_COLORS = [
+  { main: '#1a914e', alt: '#c8edd1' },
+  { main: '#306c72', alt: '#dcf3f9' },
+  { main: '#b56d48', alt: '#f9d5c0' },
+  { main: '#536491', alt: '#d4dbe8' }
+]
 
-  return 'critical'
-}
-
-const PeopleGraph = ({ amount }) => {
-  const peeps = [...Array(Math.floor(amount / 1000))]
+// TODO replace placeholder 20000 max with the top value from the street?
```

Comment:
What's this placeholder comment referring to?

Reply:
This is leftover from a previous commit ✂️